openshift-container-platform-4/policies/AC-Access_Control/component.yaml

```diff
@@ -412,6 +412,20 @@
         the separation of information flows, and the mechanisms and
         techniques used to accomplish this separation.
 
+- control_key: AC-5
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        AC-5 is an organizational control outside the scope of OpenShift
+        configuration.
+        
+        Note OpenShift has authorization mechanisms within the product to
+        support the organizational controls.
+        
```

Comment:
Nice! One small change needed prior to merge is to breakout the subcontrol responses. For example, from AC-1:

https://github.com/ComplianceAsCode/redhat/blob/master/openshift-container-platform-4/policies/AC-Access_Control/component.yaml#L13#L23

`````
- control_key: AC-1
  standard_key: NIST-800-53
  covered_by: []
  implementation_status: not applicable
  narrative:
    - key: a
      text: |
        AC-1(a) is an organizational control outside the scope of OpenShift configuration.
    - key: b
      text: |
        AC-1(b) is an organizational control outside the scope of OpenShift configuration.
`````

When automated tooling auto generates FedRAMP documents, SSP templates, and OSCAL, the ``key`` subelements make sure the text gets placed into the right locations. 

So for this control, the code would look something like....

`````
- control_key: AC-5
  standard_key: NIST-800-53
  covered_by: []
  implementation_status: not applicable
  narrative:
    - key: a
      text: |
        Organizational processes to separate the organization-defined duties of individuals
        are outside the scope of OpenShift configuration.
    - key: b
      text: |
        Organizational processes to document the separation of duties of individuals
        are outside the scope of OpenShift configuration.
    - key: c
      text: |
        Organizational processes to define information system access authorizations to
        support separation of duties are outside the scope of OpenShift configuration.

        However, to aide in such processes, note that OpenShift provides the capability to
        implement Role-based access control (RBAC) objects which determine whether a
        user is allowed to perform a given action. Documentation on the OpenShift RBAC
        capabilities can be found at
        https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html
`````

Reply:
Made those changes.

Reply:
Nice! Resolving.